ocfweb/main/openbugbounty.py

```diff
@@ -0,0 +1,11 @@
+from django.http import HttpResponse
+
+SECURITY_TXT = """\
+OpenBugBounty: https://openbugbounty.org/bugbounty/ucbocf/
+"""
```

Comment:
can we also include something like:

```
Contact: security@ocf.berkeley.edu
```

in case people would like to casually report issues without using OpenBugBounty? it's good to have OpenBugBounty as an option, but presumably we don't want to make people jump through hoops if they'd rather just send a message.